ios/ZulipMobile/Release.xcconfig

```diff
@@ -0,0 +1,8 @@
+//
+//  Release.xcconfig
+//  ZulipMobile
+//
+//  This file is in .gitignore. Add your own key to DEVELOPMENT_TEAM
+
+CODE_SIGN_IDENTITY = iPhone Developer
+DEVELOPMENT_TEAM =
```

Comment:
Yeah that's a good idea. I was looking for a way to check it in but then have git ignore future changes to it but doesn't seem like that behavior is really possible